tests/process.test.ts

```diff
@@ -0,0 +1,47 @@
+import _process from "process";
+
+describe("process", () => {
+  it("should have a process env", () => {
+    assert.equal(_process.env.PWD, process.env.PWD);
```

Comment:
Will this ENV exist for all platforms? Can we check for `_` or something else?

Reply:
I can do assertion simply by `process.env` memory reference, should be sufficient.